apps/checkout-app/frontend/components/elements/AppContainer/styles.tsx

```diff
@@ -1,7 +1,7 @@
-import { makeStyles } from "@saleor/macaw-ui";
+import { makeStyles, SaleorTheme } from "@saleor/macaw-ui";
 
 export const useStyles = makeStyles(
-  (theme) => ({
+  (theme: SaleorTheme) => ({
```

Comment:
Shouldn't this be already typed coming from macaw-ui?

Reply:
It should. I run tsc, it failed there, I added SaleorTheme, it worked, I removed it again, it still works. A little bit weird 🤨. Anyway, I removed it.